import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java

```diff
@@ -610,7 +610,7 @@ private void clearUsedValues(){
 				Node langAttr = elementsByTagName.item(i).getAttributes().getNamedItem("xml:lang");
 				if (langAttr == null)
 					continue;
-				String lang = langAttr.getNodeValue();
+				String lang = langAttr.getNodeValue().trim().toLowerCase();
 				if (!list.contains(lang)) {
```

Comment:
extractLanguage method is the same in both processors, please lets take this opportunity to move the method to IATIProcessor base class